osgeo_importer/views.py

```diff
@@ -79,40 +86,50 @@ class FileAddView(FormView, ImportHelper, JSONResponseMixin):
     json = False
 
     def form_valid(self, form):
-        upload = UploadedData(user=self.request.user)
+        upload = UploadedData.objects.create(user=self.request.user)
```

Comment:
I can't tell why this needed to change, but my understanding is that the whole point of create() is to save() without having to go and write a whole extra line with a save() call in it. I'd be interested in a correction here though